### PR TITLE
[@types/ember] Add `currentRouteName` to Ember RouterService

### PR DESCRIPTION
**Pull request by lsg-braymon**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://www.emberjs.com/api/ember/3.3/classes/RouterService/properties/currentRouteName?anchor=currentRouteName
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/ember/index.d.ts [hunk 1]
```diff
@@ -2111,6 +2111,10 @@ declare module 'ember' {
              * supplied callback function using `this.route`.
              */
             static map(callback: (this: RouterDSL) => void): void;
+            /**
+             * This property represent the logical name of the route, which is dot separated.
+             */
+            currentRouteName: string;
```
Comment (mike-north) [inline types/ember/index.d.ts]: Can we have this as a readonly property, in alignment with the implementation

https://github.com/emberjs/ember.js/blob/b08b93c925bbc7fd4c39ad94cc8280abc56946e0/packages/ember-routing/lib/services/router.js#L41

**Review by mike-north — CHANGES_REQUESTED** [review]
Thanks for your contribution to Ember's type definitions!

In addition to the `readonly` thing, please also ensure this has test coverage. in the ./tests folder, find a reasonable place to add positive test case (i.e., 

```ts
const x: string = routerService.currentRouteName;
```

**Comment by lsg-braymon** [comment]
@mike-north Thanks for the feedback.
Apologies for my original changes wherein I placed the `currentRouteName` in the Router class instead of the RouterService. I also added `currentURL` in the updated version.

**Review by mike-north — APPROVED** [review]
LGTM. :shipit:

**Pull request merged**